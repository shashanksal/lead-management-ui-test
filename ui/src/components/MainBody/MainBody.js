import React from "react";

//Material UI Components
import Paper from "@mui/material/Paper";
import Grid from "@mui/material/Grid";
import Box from "@mui/material/Box";
import Container from "@mui/material/Container";

// Styles
import { styled } from "@mui/material/styles";

// Custom components import
import Tabs from "../TaskTabs";

import { THEME_COLOUR_2 } from "../../utils/constants";

const BackgroundPaper = styled(Paper)(({ theme }) => ({
	...theme.typography.body2,
	color: theme.palette.text.secondary,
	height: "100%",
	backgroundColor: THEME_COLOUR_2
}));

const MainBody = () => {
	return (
		<Container maxWidth="md">
			<Box m={1}>
				<Grid container spacing={2}>
					<Grid item xs={12}>
						{/* <Box
							sx={{
								width: "100%",
								height: "100%"
							}}
						> */}
						<BackgroundPaper elevation={0}>
							<Tabs />
						</BackgroundPaper>
						{/* </Box> */}
					</Grid>
				</Grid>
			</Box>
		</Container>
	);
};

export default MainBody;
