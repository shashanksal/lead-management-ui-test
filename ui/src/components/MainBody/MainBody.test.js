import React from "react";
import { render } from "@testing-library/react";

import MainBody from "./MainBody";

describe("Mainbody", () => {
	it("should match snapshot", () => {
		const { asFragment } = render(<MainBody />);
		expect(asFragment()).toMatchSnapshot();
	});
});
