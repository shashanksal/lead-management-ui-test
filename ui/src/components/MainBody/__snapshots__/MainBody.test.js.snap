// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mainbody should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="MuiContainer-root MuiContainer-maxWidthMd css-xdmu94-MuiContainer-root"
  >
    <div
      class="MuiBox-root css-1ynyhby"
    >
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 css-mhc70k-MuiGrid-root"
      >
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-1em9ut6-MuiGrid-root"
        >
          <div
            class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation0 css-1k0y9i-MuiPaper-root"
          >
            <div
              class="MuiGrid-root css-vj1n65-MuiGrid-root"
            >
              <div
                class="MuiBox-root css-8atqhb"
              >
                <div
                  class="MuiTabs-root css-1ujnqem-MuiTabs-root"
                >
                  <div
                    class="MuiTabs-scroller MuiTabs-fixed css-jpln7h-MuiTabs-scroller"
                    style="overflow: hidden; margin-bottom: 0px;"
                  >
                    <div
                      aria-label="myapp tabs"
                      class="MuiTabs-flexContainer MuiTabs-centered css-1wf8b0h-MuiTabs-flexContainer"
                      role="tablist"
                    >
                      <button
                        aria-selected="true"
                        class="MuiButtonBase-root MuiTab-root MuiTab-textColorPrimary MuiTab-fullWidth Mui-selected css-pabxj7-MuiButtonBase-root-MuiTab-root"
                        role="tab"
                        tabindex="0"
                        type="button"
                      >
                        Invited
                      </button>
                      <button
                        aria-selected="false"
                        class="MuiButtonBase-root MuiTab-root MuiTab-textColorPrimary MuiTab-fullWidth css-pabxj7-MuiButtonBase-root-MuiTab-root"
                        role="tab"
                        tabindex="-1"
                        type="button"
                      >
                        Accepted
                      </button>
                    </div>
                    <span
                      class="MuiTabs-indicator css-1aquho2-MuiTabs-indicator"
                      style="left: 0px; width: 0px; background: rgb(255, 122, 18);"
                    />
                  </div>
                </div>
              </div>
              <div
                class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
                style="padding-top: 24px;"
              >
                <div
                  class="job-card-wrapper MuiBox-root css-0"
                >
                  <div
                    class="MuiTypography-root MuiTypography-h3 css-gepadz-MuiTypography-root"
                  >
                    No new jobs to show
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
